source/edge-server.txt

```diff
@@ -79,28 +80,26 @@ Edge Server Setup
 
 .. procedure::
 
-   .. step:: Create a Device Sync-Enabled App
+   .. step:: Create an Edge Server Service
 
-      Create an :ref:`App Services App <create-app>` with 
-      :ref:`Device Sync enabled <enable-sync>`. To get started quickly, 
-      :ref:`create a template app <create-template-app>`.
+      You can use Edge Server with MongoDB Drivers and tools in an Edge 
+      Server Service. Or you can use Edge Server with Atlas Device SDK to
+      develop mobile and edge applications with Device Sync, and get the
+      benefits of conflict resolution and roaming between servers.
 
-      Once your App is created, coordinate with your Product or Account 
-      Representative to enable Edge Server.
+      To get started quickly, 
+      :ref:`create a template app <create-template-app>`.
```

Comment:
it seems weird to just send users off to the Template Apps page without further context, esp since the destination doesn't mention Edge Server anywhere. Can we bolster this with some additional instruction for someone who isn't familiar with our template apps? e.g. `To get started quickly using an Atlas Device SDK mobile app... `